components/feature/downloads/src/main/java/mozilla/components/feature/downloads/manager/FetchDownloadManager.kt

```diff
@@ -41,11 +44,19 @@ class FetchDownloadManager<T : AbstractFetchDownloadService>(
 
     private var isSubscribedReceiver = false
 
-    override val permissions = if (SDK_INT >= P) {
-        arrayOf(INTERNET, WRITE_EXTERNAL_STORAGE, FOREGROUND_SERVICE)
-    } else {
-        arrayOf(INTERNET, WRITE_EXTERNAL_STORAGE)
-    }
+    // Do not require WRITE_EXTERNAL_STORAGE permission on API 29 and above (using scoped storage)
+    override val permissions
```

Comment:
For we use this download manager that is based of [GeckoView ](https://mozilla.github.io/geckoview/) :)

Reply:
Ok